tests/test_other.py

```diff
@@ -503,10 +503,10 @@ def test_emcc_asm_v_wasm(self):
   def test_emcc_cflags(self):
     output = self.run_process([EMCC, '--cflags'], stdout=PIPE)
     flags = output.stdout.strip()
-    self.assertContained(shared.shlex_join(shared.emsdk_cflags([])), flags)
+    self.assertContained('-target wasm32-unknown-emscripten', flags)
```

Comment:
Is this the same with the return value of `emsdk_cflags`? `emsdk_cflags` looks like it can return a lot more flags.. The same for 3 lines below.

Reply:
Yeah I agree checking only the target flag should be sufficient.